MantidQt/MantidWidgets/src/DataProcessorUI/QDataProcessorOneLevelTreeModel.cpp

```diff
@@ -197,6 +187,33 @@ bool QDataProcessorOneLevelTreeModel::setData(const QModelIndex &index,
   return true;
 }
 
+/** Sets the currently highlighted row
+* @param position : The position of the row to be highlighted
+* @param parent : The parent of this row
+* @return : Boolean indicating whether the row was successfully highlighted
+*/
+bool QDataProcessorOneLevelTreeModel::addHighlighted(
+    int position, const QModelIndex &parent) {
+
+  if (parent.isValid())
```

Comment:
Items in `QDataProcessorOneLevelTreeModel` are expected to only be a single list of top-level items. If the index was valid for a parent item, it would imply that a parent item exists (which should not be possible since there are no parent items holding child items).